plugin-catalog/src/components/plugins/Detail.tsx

```diff
@@ -118,6 +118,21 @@ export interface PurePluginDetailProps {
   onAlertClose: () => void;
 }
 
+const pluginActionSnackbar = (closeCallback: () => void) => {
+  return (
+    <>
+      <Button
+        onClick={() => {
+          window.location.reload();
+        }}
+      >
+        Reload Now
+      </Button>
```

Comment:
I think this may clash with what the alertMessage may be, so it feels like something we need to pass to the PurePluginDetail...

Reply:
Updated to be rendered only for install and uninstall alert messages

Reply:
In the end, I changed it to almost what you had originally. I think the code base is confusing, but we can address that later.